drivers/serial/uart_nrf_sw_lpuart.c

```diff
@@ -1026,7 +1026,11 @@ static const struct uart_driver_api lpuart_api = {
 #endif
 };
 
-DEVICE_DEFINE(lpuart, "LPUART", lpuart_init, device_pm_control_nop,
+BUILD_ASSERT(DT_IRQ(DT_INST(0, nordic_nrf_gpiote), priority) ==
+	     DT_IRQ(DT_PARENT(DT_INST(0, nordic_nrf_sw_lpuart)), priority),
```

Comment:
```suggestion
	     DT_IRQ(DT_PARENT(DT_NODELABEL(lpuart)), priority),
```
for consistency with: https://github.com/nrfconnect/sdk-nrf/blob/0575cab37f7569d8a8feed89d5ddf4545b212be5/drivers/serial/uart_nrf_sw_lpuart.c#L1033
Perhaps it would be better to also use `DT_NODELABEL(gpiote)` instead of `DT_INST(0, nordic_nrf_gpiote)` above.